.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"

```

Comment:
package manager is yarn

Reply:
@amandesai01 give a read here: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#package-ecosystem

but i think it's handle by "npm"

Reply:
makes sense. went through the link.